Source/Core/Core/IOS/Network/WD/Command.cpp

```diff
@@ -8,29 +8,415 @@
 #include <cstring>
 #include <string>
 
+#include "Common/BitSet.h"
 #include "Common/CommonTypes.h"
 #include "Common/Logging/Log.h"
 #include "Common/Network.h"
 #include "Common/Swap.h"
-
 #include "Core/HW/Memmap.h"
 #include "Core/IOS/Network/MACUtils.h"
 
 namespace IOS::HLE::Device
 {
+namespace
+{
+constexpr u16 LegalChannelMask = 0b111111111111110u;
+constexpr u16 LegalNitroChannelMask = 0b11111111111110u;
```

Comment:
I didnt see those differed by a one, could you please fill them up to the next multiple of 8, and/or use digit separators?

Reply:
Sure